client/components/App.jsx

```diff
@@ -1,17 +1,60 @@
 import React from 'react';
+import axios from 'axios';
 import Calendar from './Calendar';
+import Guest from './Guest';
+import ReserveButton from './ReserveButton';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      listingInfo: [],
+    };
   }
 
+  componentDidMount() {
+    this.getListings();
+  }
+
+  getListings() {
+    axios.get('/listings')
```

Comment:
Something small we can go over late, but to make it easier for when we do proxy server will need to eventually refactor this so it's the same as everyone else's '/rooms/id' but dont worry about that since you in crunch time haha